http/pool.go

```diff
@@ -32,220 +32,295 @@ import (
 	"github.com/skydive-project/skydive/config"
 )
 
-// WSClientPool represents a pool of WebSocket clients
-type WSClientPool struct {
+type WSSpeakerPool interface {
+	AddClient(c WSSpeaker) error
+	AddEventHandler(h WSSpeakerEventHandler)
+	GetClients() []WSSpeaker
+	GetConnectedClient() WSSpeaker
```

Comment:
PickConnectedClient ?

Something that clearly states that a client is randomly chosen